packages/varlet-cli/site/pc/components/LogoAnimation.vue

```diff
@@ -48,13 +55,10 @@ export default defineComponent({
         await nextTick()
       }
       clearTimeout(resetTimer)
-
-      const newBRect = animationEl.value?.getBoundingClientRect()
-      if (newBRect) {
-        resetTimer = window.setTimeout(() => {
-          proxyRect.value = newBRect
-        }, 200)
-      }
+      resetTimer = window.setTimeout(() => {
+        const newBRect = animationEl.value?.getBoundingClientRect()
+        if (newBRect) proxyRect.value = pick(newBRect, 'width', 'height', 'left', 'top')
+      }, 200)
```

Comment:
You only need to change one line here `proxyRect.value = newBRect`

Reply:
I think this can have better performance? `getBoundingClientRect` may cause browser reflow

Reply:
My point is whether it is necessary to execute setTimeout without resize.